### PR TITLE
Add randomized load test

### PR DESCRIPTION
**Pull request by gmarek**

### CODE REVIEW

File: test/e2e/load.go [hunk 1]
```diff
@@ -63,6 +63,16 @@ const (
 	nodeCountPerNamespace = 100
 )
 
+var randomKind = schema.GroupKind{Kind: "Random"}
+
+var knownKinds = []schema.GroupKind{
+	api.Kind("ReplicationController"),
+	extensions.Kind("Deployment"),
+	// TODO: uncomment when Jobs are fixed
```
Comment (wojtek-t) [inline test/e2e/load.go]: What do you mean by "Jobs are fixed"? Can you add issue number?

**Comment by gmarek** [comment]
@k8s-bot unit test this issue: #41768

**Comment by gmarek** [comment]
@k8s-bot unit test this issue: #41768

**Comment by gmarek** [comment]
@k8s-bot unit test this issue: #41768

**Comment by gmarek** [comment]
@k8s-bot gce etcd3 e2e test this

**Pull request merged**